variables.json

```diff
@@ -0,0 +1,4 @@
+{
+  "variable_name": "variable_value",
+  "policy-server-component": "PolicyServer"
```

Comment:
Quibble: Maybe better to name the variable `policy-server-customresource`, or `policy-server-cr` (written PolicyServer), in preparation for having a `policy-server-image`, or `policy-server-pod`, or `policy-server-binary` (written policy-server).

Reply:
It's just an example to show how it works, constructing the list of variable_names and variable_values will be quite an intensive joint effort. I think I'll move this to draft, as a reminder to remove the changes to quick-start.md before merging. They come after, the name and values are decided.

Reply:
fair enough, no need to block on this for this pr. Resolving.